web/crux/src/interceptors/prisma-error-interceptor.ts

```
@@ -23,13 +24,18 @@ export default class PrismaErrorInterceptor implements NestInterceptor {
 
   onError(_context: ExecutionContext, err: Error): any {
     if (err instanceof Prisma.PrismaClientKnownRequestError) {
-      this.handlePrismaError(err)
+      const exception = PrismaErrorInterceptor.transformPrismaError(err)
+      if (exception != null) {
+        throw exception
+      }
+
+      return
```

Why return? Is there a case where we swallow the exception?